### ISSUE TITLE
mapTrackerToProps doesn't re-evaluates when props changed on Meteor 1.4.1

### ISSUE THREAD

**Issue by tempik** [lhz516/react-redux-meteor#1]
>  If ownProps is specified as a second argument, its value will be the props passed to your component, and mapTrackerToProps will be additionally re-invoked whenever the component receives new props (e.g. if props received from a parent component have shallowly changed, and you use the ownProps argument, mapTrackerToProps is re-evaluated).

**mapTrackerToProps** is calling only once on component mount ( and when reactive data source changes), but not after container component props changed. I found this issue when passing params to container component from React Router, and then i tried to get it working by direct props setup from parent component by click on button, ineffectually.

**Comment by tempik** [comment]
i'd like to note that the **mapStateToProps** called every time i changed props from parent component as it should works.

**Comment by lhz516** [comment]
Thanks for letting me know. I will try to fix this bug soon.

**Comment by MaxTwentythree** [comment]
any news on this?

**Comment by john-osullivan** [comment]
@lhz516 , is this library actively maintained?  No worries if not, just want to check because the repo is many commits behind main redux and I don't want to pick up a dependency that will go stale.

**Comment by lhz516** [comment]
Thanks for everyone but this repo is no longer maintained, because I'm currently using [react-komposer](https://github.com/arunoda/react-komposer) which is great for Meteor + React.

If anyone wants to take over this project, I can transfer the ownership to you.

**Comment by john-osullivan** [comment]
Nope, not looking to pick up any more work than I've already got! How are
you using react-komposer to bridge Minimongo & Redux? That's my big issue
right now. Amy helpful blog posts you could send me toward?

**Comment by lhz516** [comment]
@john-osullivan https://github.com/arunoda/react-komposer#for-meteors-tracker

Read the complete README. Here is the usage of how react-komposer works with Redux and Tracker(which includes the reactive mini mongo)

**Comment by LaughingBubba** [comment]
@lhz516 I'll take over the project if you want.

**Comment by lhz516** [comment]
@LaughingBubba Good to know. Instead of transfering the ownership of this project, I recommend you to create your own repo.

You can implement with the latest `react-redux`. Once you've done, I can unpublish `react-redux-meteor` on NPM so that you can reuse the package name to maintain it.

**Comment by john-osullivan** [comment]
FWIW, I found that [meteor-redux-middlewares](
https://github.com/samybob1/meteor-redux-middlewares) got the job done.
Maybe it doesn't fit your needs, but there something to be said for
coalescing this Meteor+React+Redux community around some common options.
